projects/admin-core/assets/locale/messages.admin-core.nl.xlf

```diff
@@ -379,8 +379,8 @@
         <target>Terrein</target>
       </trans-unit>
       <trans-unit id="admin-core.application.terrain-layers" datatype="html">
-        <source>Terrain Layers</source>
-        <target>Terrein Lagen</target>
+        <source>Terrain layers</source>
+        <target>Terrein lagen</target>
```

Comment:
In het Nederlands moeten samenstellingen aan elkaar geschreven worden (spatiegebruik.nl)